libr/asm/arch/arc/gnu/arc-opc.c

```diff
@@ -4504,7 +4504,7 @@ arc_opcode_lookup_suffix (const struct arc_operand *type, int value)
     }
 
   /* ??? This is a little slow and can be speeded up.  */
-    for (v = arc_suffixes, end = arc_suffixes + arc_suffixes_count; v < end; ++v) {
+    for (v = arc_suffixes, end = arc_suffixes + arc_suffixes_count; v < end; v++) {
```

Comment:
I didnt updated this one because its from gnu

Reply:
ok 2258c7faa95fc2c19e1fee59d3d1ffe8cc8297cd